File: learning/challenges/ex1/ex1.py
# -*- coding: utf-8 -*-

def multiplesOf(number):
	multiplys = []

	i = 1
	while i < number:
		if i % 3 == 0 or i % 5 == 0:
			multiplys.append(i)
		i += 1

	return sum(multiplys)

def sum(multiplys):
	suma = 0

	for d in multiplys:
		suma = suma + d

	return suma

print multiplesOf(10)

File: learning/challenges/ex1/ex1.js
function multiplesOf(number) {
	var multiplys = [];
	for (var i = 0; i < number; i++){
		if ( i % 3 == 0 || i % 5 == 0) {
			multiplys.push(i);
		}
	}
	console.log(multiplys);
	return sum(multiplys);
}

function sum(multiplys) {
	var suma = 0;
	for (var i = 0; i < multiplys.length; i++) {
		sum = sum + multiplys[i];
	}

	return suma;
}

// Uncomment the new line for test
// console.log(multiplesOf(100))
